### ISSUE TITLE
Position of name and value makes it difficult to quickly read metatag values.

### ISSUE THREAD

**Issue by mikeethedude** [jonasmerlin/astro-seo#84]
I'm willing to submit a pull request for this.

Right now the extended <meta> tags (`src/components/ExtendedTags.astro`) output from this project output with the content before the name and that makes it difficult to parse at a glance.

```<meta
      content={content}
      http-equiv={httpEquiv}
      media={media}
      name={name}
      property={property}
    />
```
I'd think it would make it easier if the name and property were the first attributes filled there.

**Comment by jonasmerlin** [comment]
@mikeethedude Thank you for your input! Since this input seems to stem from your experience with using this feature in your project(s), I would be more than happy to merge a pull request improving ergonomics this way. Please feel free to go ahead with this, and I'll put it into the next version!

**Comment by mikeethedude** [comment]
Awesome! Pull request is open. I think this is a pretty simple swap on properies. See https://github.com/jonasmerlin/astro-seo/pull/85

**Comment by jonasmerlin** [comment]
Landed in 0.8.1
